cinema_service/urls.py

```diff
@@ -1,6 +1,8 @@
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
Consider making the inclusion of `debug_toolbar_urls` conditional based on the `DEBUG` setting to avoid exposing it in production environments.